src/utils/gaa.js

```diff
@@ -2045,13 +2064,18 @@ export class GaaMetering {
   }
 
   static newUserStateToUserState(newUserState) {
+    // Convert registrationTimestamp
+    const registrationTimestamp = convertPotentialTimestampToSeconds(
```

Comment:
I feel like we should enforce the timestamp provided to be a certain range, but lets maybe name this `registrationTimestampSeconds` just to be clear

Reply:
Sounds good, thanks!